libs/api/domains/auth/src/lib/meDelegations.service.ts

```diff
@@ -0,0 +1,94 @@
+import { Injectable } from '@nestjs/common'
+
+import { Auth, AuthMiddleware, User } from '@island.is/auth-nest-tools'
+import {
+  DelegationDTO,
+  UpdateDelegationScopeDTO,
+  MeDelegationsControllerFindAllDirectionEnum,
+  MeDelegationsApi,
+} from '@island.is/clients/auth-public-api'
+
+import {
+  UpdateDelegationInput,
+  DeleteDelegationInput,
+  DelegationInput,
+  CreateDelegationInput,
+} from './dto'
+import { DelegationByOtherUserInput } from './dto/delegationByOtherUser.input'
+
+@Injectable()
+export class MeDelegationsService {
+  constructor(private delegationsApi: MeDelegationsApi) {}
+
+  private delegationsApiWithAuth(auth: Auth) {
+    return this.delegationsApi.withMiddleware(new AuthMiddleware(auth))
+  }
+
+  getDelegations(user: User): Promise<DelegationDTO[]> {
+    return this.delegationsApiWithAuth(user).meDelegationsControllerFindAll({
+      direction: MeDelegationsControllerFindAllDirectionEnum.Outgoing,
+      isValid: true,
```

Comment:
Fyi. You should also see delegations that are valid in the future. Not sure if this encompasses that or if it should be a `valid` string enum filter instead.